.github/workflows/main.yml

```diff
@@ -113,6 +113,16 @@ jobs:
         RUSTFLAGS='-C target-feature=+atomics,+bulk-memory,+mutable-globals' \
           cargo test --target wasm32-unknown-unknown --test headless -Z build-std=std,panic_abort
 
+  test_node:
+    name: "Run wasm-bindgen node tests"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - run: rustup update --no-self-update stable && rustup default stable
+    - run: rustup target add wasm32-unknown-unknown
+    - run: |
+          cargo test --target wasm32-unknown-unknown --test headless_no_threads -p js-sys
```

Comment:
Why does this require a separate job?
It should be covered here:
https://github.com/rustwasm/wasm-bindgen/blob/4596d4a633900a3f474c715aa51937ed0282e205/.github/workflows/main.yml#L188-L202